vortex-layout/src/layouts/struct_/eval_expr.rs

```diff
@@ -50,6 +50,7 @@ impl ExprEvaluator for StructReader {
         .into_array();
 
         // Recombine the partitioned expressions into a single expression
+        // use vortex_array::ArrayDType;
```

Comment:
dead code?

Reply:
nope, useful for debugging, this is going into @gatesn mega PR